webapp/src/app.tsx

```diff
@@ -4887,7 +4925,8 @@ export class ProjectView
             this.editor != this.blocksEditor ? "editorlang-text" : "",
             this.editor == this.textEditor && this.state.errorListState,
             'full-abs',
-            pxt.appTarget.appTheme.embeddedTutorial ? "tutorial-embed" : ""
+            pxt.appTarget.appTheme.embeddedTutorial ? "tutorial-embed" : "",
+            // isMultiplayerGame ? "multiplayer-preview" : "",
```

Comment:
needed?

Reply:
Ah, good catch, I had set this class for just a minute while making fullscreensim layout correctly but was able to move it down to just be on sidepanel, will remove